starwars/src/components/Card.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styled from "styled-components";
+
+const Container = styled.div`
+  background-color: white;
+  max-width: 600px;
+  padding: 5px 10px;
+  margin: 15px auto;
+  border-radius: 10px;
+  opacity: 0.8;
+  box-sizing: border-box;
+  text-align: center;
+
+  & h1 {
```

Comment:
If you were trying to next the `h1` tag, you don't need the `&` character. Nesting with styled-components is very similar to LESS.